pkg/cache/v3/linear.go

```diff
@@ -94,45 +125,64 @@ func WithLogger(log log.Logger) LinearCacheOption {
 	}
 }
 
+// WithComputeStableVersions ensures the cache tracks the resources stable versions from the beginning,
+// avoiding the first watch stalling until the computation has been done on all resources.
+func WithComputeStableVersions() LinearCacheOption {
```

Comment:
Could you add a comment explaining why we wouldn't want to enable this option, or is it recommended to always enable this, and this is not the default for backwards compatibility reasons?